source/standards/threat-modelling.html.md.erb

```diff
@@ -55,11 +58,12 @@ aspects and make sure we get a rounded view of all threats.
 The purpose of this exercise is to talk through the service, architecture or the
 user journey with the people in the session, allowing for questions and answers.
 
-In-person, diagrams are typically drawn up on a whiteboard, followed by
-questions and post-its of potential threats stuck at relevant points.
+Remotely, the typical exercise, [Jamboard] can be used to add virtual post-its
```

Comment:
There are accessibility concerns about Jamboard so not sure we should be recommending it. Could we put something like 'you can use digital interactive whiteboard software to add...'?

Reply:
I am keen to list out specific tools. I expect Google to be top of the game, so I can't imagine there's anything more accessible. Any suggestions for alternatives?

Would it help to add something like "if your team has not used Jamboard, test it with your team to make sure it is accessible" ?

Reply:
The GDS accessibility community are active on Slack. They would be the best people to ask, but I think Mural is generally considered more accessible than Jamboard and many teams seem to be using it at the moment.

Reply:
This one is hopefully resolved